frontend/src/screens/MigrateNodeSuccess.tsx

```diff
@@ -2,12 +2,12 @@ import { BadgePlus, PowerOff, Save } from "lucide-react";
 import AppHeader from "src/components/AppHeader";
 import { Card, CardContent } from "src/components/ui/card";
 
-export function BackupNodeSuccess() {
+export function MigrateNodeSuccess() {
```

Comment:
I am wondering about the name of this component and file. Maybe it should be `CreateNodeMigrationFileSuccess`